pkg/eval/builtin_fn_fs.go

```diff
@@ -40,21 +39,6 @@ var ErrStoreNotConnected = errors.New("store not connected")
 // ▶ [&path=/Users/foo/.elvish &score=96.79928]
 // ```
 
-//elvdoc:fn path-\*
```

Comment:
You should not remove the documentation for these builtins. You're deprecating them in this commit; not removing them.